pkg/assets/handlers.go

```diff
@@ -155,20 +154,20 @@ type WebConsoleVersion struct {
 	OpenShiftVersion  string
 }
 
-var extensionPropertiesTemplate = html_template.Must(html_template.New("webConsoleExtensionProperties").Parse(`
+var extensionPropertiesTemplate = template.Must(template.New("webConsoleExtensionProperties").Parse(`
 window.OPENSHIFT_EXTENSION_PROPERTIES = {
-  {{ $values := .ExtensionPropertyValues }}
-  {{ range $i, $key := .ExtensionPropertyKeys }}
-  {{ if $i }},{{ end }}
-  {{ $value := index $values $i }}
-  "{{ $key }}": "{{ $value }}"
-  {{ end }}
+{{ range $i, $property := .ExtensionProperties }}{{ if $i }},{{ end }}
+  "{{ $property.Key | jsstr }}": "{{ $property.Value | jsstr }}"{{ end }}
```

Comment:
`| js` seems to do what we want. It was the additional inferred filters in the html/template package that was making the output weird